### PR TITLE
Parallel acceptance tests

### PR DESCRIPTION
**Pull request by NeosinneR**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -26,6 +26,8 @@ lang
 .mp_svn
 /nbproject/
 tests/_data/acceptanceGenerated.sql
+tests/acceptance/_groups/*
+!tests/acceptance/_groups/.gitkeep
```
Comment (JanJakes) [inline .gitignore]: We could do this as in `tests/acceptance/_output` - a `.gitignore` file in the `_groups` directory containing:
```
*
!.gitignore
```

File: RoboFile.php [hunk 2]
```diff
@@ -532,4 +535,12 @@ protected function getPluginVersion($file) {
...
     return !empty($m[1]) ? trim($m[1]) : false;
   }
+
+  public function parallelSplitTests() {
+    return $this->taskSplitTestFilesByGroups(4)
+      ->projectRoot('.')
+      ->testsFrom('tests/acceptance')
+      ->groupsTo('tests/acceptance/_groups/group_')
+      ->run();
+  }
```
Comment (JanJakes) [inline RoboFile.php]: Could this be called `test:acceptance:create-groups` or something like that? The commands are namespaced:
```
Available commands:
  help                        Displays help for a command
  install
  list                        Lists commands
  makepot
  packtranslations
  publish
  pushpot
  qa
  txinit
  update
  watch
 compile
  compile:all
  compile:css
  compile:js
 container
  container:dump
 delete
  delete:docker
 qa
  qa:code-sniffer
  qa:lint
  qa:lint-javascript
  qa:phpstan
 security
  security:composer
 svn
  svn:checkout
  svn:publish
 test
  test:acceptance
  test:acceptance-multisite
  test:coverage
  test:debug-integration
  test:debug-unit
  test:failed-integration
  test:failed-unit
  test:integration
  test:javascript
  test:multisite-integration
  test:unit
 watch
  watch:css
  watch:js
```

File: tests/acceptance/SaveNewsletterAsDraftCest.php [hunk 3]
```diff
@@ -30,14 +30,15 @@ function saveStandardNewsletterAsDraft(\AcceptanceTester $I) {
...
+
     // step 4 - Choose list and send
     $send_form_element = '[data-automation-id="newsletter_send_form"]';
     $I->waitForElement($send_form_element);
     $I->seeInCurrentUrl('mailpoet-newsletters#/send/');
     $I->selectOptionInSelect2('WordPress Users');
 
     $I->click('Save as draft and close');
-    $I->waitForText($newsletter_title, 5, '[data-automation-id^="listing_item_"]');
+    $I->waitForText($newsletter_title, 30);
```
Comment (JanJakes) [inline tests/acceptance/SaveNewsletterAsDraftCest.php]: This is kind of in conflict with what Alex was working on. Basically the idea is to never use "magic" timeouts in tests and leave it to the default 10s. If we need longer timeout in some, then there would be a single constant for that. But here - is really 10s not enough?

File: tests/acceptance/SubscribeToMultipleListsCest.php [hunk 4]
```diff
@@ -12,6 +12,13 @@
 require_once __DIR__ . '/../DataFactories/Segment.php';
 
 class SubscribeToMultipleListsCest {
+
+  function _before(\AcceptanceTester $I) {
+    // Clean table with subscriber IPs, so another test
+    // running in parallel won't block this subscription
```
Comment (JanJakes) [inline tests/acceptance/SubscribeToMultipleListsCest.php]: I guess the comment is wrong. A test running in parallel cannot affect this test in any way. I think we said it's due to some previous test if I'm not mistaken.

File: .gitignore (hunk 1 above)
Comment (NeosinneR) [reply .gitignore]: Done

File: RoboFile.php (hunk 2 above)
Comment (NeosinneR) [reply RoboFile.php]: Done, `test:acceptance-group-tests` - I don't think there can be `:` twice

File: tests/acceptance/SaveNewsletterAsDraftCest.php (hunk 3 above)
Comment (NeosinneR) [reply tests/acceptance/SaveNewsletterAsDraftCest.php]: Removed timeout completely

File: tests/acceptance/SubscribeToMultipleListsCest.php (hunk 4 above)
Comment (NeosinneR) [reply tests/acceptance/SubscribeToMultipleListsCest.php]: Correct, I changed it

**Pull request merged**